sql-cli/sql_cli/__main__.py

```diff
@@ -37,20 +38,19 @@ def about() -> None:
 
 @app.command(help="Generate the Airflow DAG from a directory of SQL files.")
 def generate(
-    directory: Path = typer.Argument(
+    workflow_directory: Path = typer.Argument(
```

Comment:
The idea was that this would be `workflow_name` (only the name of the directory), and that, together with the `project_dir` we can build the path to it.